core/arch/arm/plat-zynqmp/platform_config.h

```diff
@@ -86,6 +86,10 @@
 #error "Unknown platform flavor"
 #endif
 
+#define CSU_DMA_BASE		0xFFC80000
```

Comment:
Official name for this is `CSUDMA`

Reply:
> > > Also this AES driver here and Linux kernel driver need to co-live together so there is no race... Should these actually be configured with device tree so that only either one can use or claim ownership -- or then the OP-TEE drops usage before Linux or U-Boot uses that resource (or uses only in between of FSBL/SPL and u-boot).
> > 
> > 
> > you probably noticed that I didnt integrate the driver with the crypto api: this is because the use case for this implementation of AES-GCM is only the encryption of the DNA string (read from eFUSE) just so we get a cryptography strong - and secret- HUK.
> 
> Please note that PS DNA is not considered stable in MPSoC. Only guarantee is given for PL DNA.

not sure what you mean, the DNA is an eFUSE (well 3 eFuses) also where did you get this information from?

Reply:
You quoted to wrong place -- I'll reply to in main comment stream.